docs/src/main/asciidoc/spring-cloud-consul.adoc

```diff
@@ -196,6 +196,78 @@ spring:
             - "Some other value"
 ----
 
+==== TTL Health Check
+
+A Consul https://www.consul.io/docs/discovery/checks#ttl[TTL Check] can be used instead of the default configured HTTP check.
+The main difference is that the application sends a heartbeat signal to the Consul agent rather than the Consul agent sending a request to the application.
+
+The interval the application uses to send the ping may also be configured. "10s" and "1m" represent 10 seconds and 1 minute respectively.
+The default is 30 seconds.
+
+This example illustrates the above (see the `spring.cloud.consul.discovery.heartbeat.*` properties in link:appendix.html[the appendix page] for more options).
+
+.application.yml
+----
+spring:
+  cloud:
+    consul:
+      discovery:
+        heartbeat:
+          enabled: true
+          ttl: 10s
+----
+
+===== TTL Application Status
+
+For a Spring Boot Actuator application the status is determined from its available health endpoint.
+When the health endpoint is not available (either disabled or not a Spring Boot Actuator application) it assumes the application is in good health.
+
+When querying the health endpoint, the root https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready-health-groups[health group] is used by default.
+A different health group can be used by setting the following property:
+
+.application.yml
+----
+spring:
+  cloud:
+    consul:
+      discovery:
+        heartbeat:
+          actuator-health-group: <your-custom-group-goes-here>
+----
+
+You can disable the use of the health endpoint entirely by setting the following property:
```

Comment:
For short examples of the form "You can XYZ by setting the following property" I saw some done [inline](https://github.com/bono007/spring-cloud-consul/commit/2e449a07ad1fd3b960333e36ca785e29f068de7b#diff-ef78d57b7c46488a24f1cd4b7760bf202de9c1c58f57f1c7ab469ef5fc2cf563R73) and some done in yaml blocks (this this one). I went w/ the block choice as it seemed to "pop out" a bit more in the final doc. What is the best practice here?